docs/tech/docker-hub.md

```diff
@@ -0,0 +1,9 @@
+# Adding a new repository to Docker Hub
+
+- Set up your own account on <https://hub.docker.com/>
+- Get an existing admin (normally SRE) to add you to the `praekeltfoundation` organisation
+- Create a new repository under the organisation for your new image
+- Add the `automation` team as a collaborator to your new repository with write access
+- Use the `travis encrypt` command locally to encrypt the `praekeltorgdeploy` Docker Hub password (from 1Password) - for example, encrypt `REGISTRY_PASS=supersecretpassword`
+- Use the `$REGISTRY_PASS` environment variable to do `docker login` in your Travis file
```

Comment:
I think it might be better to include a snippet of what it looks like in Travis or link to an existing travis file (or include a sample file with the docs)